README.md

```diff
@@ -65,39 +94,23 @@ The Dextrose generate-html command can be run with the following commands:
 
         --region, -r
             the aws region
-            
 
-Example config 
+The Dextrose clean-stories command should be run in following way:
+   
+    dextrose clean-stories path/to/your/components
 
-Native
-
-        module.exports = {
-            snapPath: path.join(__dirname, '../snaps'),    
-            platformName: "iOS",
-            ignoredStories: ["IGNORE"]
-        }
-
-Web
-
-        module.exports = {
-            snapPath: path.join(__dirname, '../snaps'),
-            platformName: "web",
-            breakpoints:[500, 1000],
-            ignoredStories: ["IGNORE"]
-        }
 
 ## Running the example
 To run the the example Pull the repo
 - Run `yarn` in the Dextrose folder
 - Run `yarn` in the snapshots folder (the snapshots folder is an example of how you would consume dextrose)
 - Run `./run-(ios | android | web).sh`
 
-note: please make sure you have an (emulator | simulator), (appium | selenium) and a packager running.
+note: please make sure you have an (emulator | simulator), and a packager running.
```

Comment:
Not sure what '(emulator | simulator)' means here

Reply:
if you are running android they have an emulator but for iOS you use simulators

Reply:
cool, makes sense